### DISCUSSION TITLE
'Extended' Windows 11 Support ?

### DISCUSSION THREAD

**Discussion by Clemens-Ratte-Polle** [ventoy/Ventoy#1330] [Ideas]
Got this feature?
Install Windows 11 On Unsupported PCs.
Latest Rufus Release Introduces 'Extended' Windows 11 Support.
Rufus to disable TPM and Secure Boot Windows 11 ISO USB.
https://www.tomshardware.com/news/rufus-creates-custom-windows-11-installs

**Comment by ventoy** [comment]
https://www.ventoy.net/en/plugin_control.html
`VTOY_WIN11_BYPASS_CHECK`

**Comment by Korb** [comment]
@Clemens-Ratte-Polle, related to https://github.com/ventoy/Ventoy/discussions/1861
